tick/optim/model/generalized_linear_model.py

```diff
@@ -0,0 +1,70 @@
+# License: BSD 3 clause
```

Comment:
This file is a duplication of  tick/optim/model/tests/generalized_linear_model.py and should be deleted